io.catenax.specialCharacteristics.measurement/1.0.0/Measurement.ttl

```diff
@@ -0,0 +1,82 @@
+#######################################################################
+# Copyright(c) 2024 BASF SE
+# Copyright(c) 2024 Bayerische Motoren Werke Aktiengesellschaft(BMW AG)
+# Copyright(c) 2024 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V.(represented by Fraunhofer ISST & Fraunhofer IML)
+# Copyright(c) 2024 German Edge Cloud GmbH & Co. KG
+# Copyright(c) 2024 Henkel AG & Co. KGaA
+# Copyright(c) 2024 Mercedes Benz AG
+# Copyright(c) 2024 Robert Bosch Manufacturing Solutions GmbH
+# Copyright(c) 2024 SAP SE
+# Copyright(c) 2024 Siemens AG
+# Copyright(c) 2024 T-Systems International GmbH
+# Copyright(c) 2024 ZF Friedrichshafen AG
+# Copyright(c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional information regarding copyright ownership.
+#
+# This work is made available under the terms of the Creative Commons Attribution 4.0 International(CC-BY-4.0) license, which is available at https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.specialCharacteristics.measurement:1.0.0#> .
```

Comment:
> @LuisRickert for folder/path naming we don't use camel case but underscores. Therefore please change your new created folder _io.catenax.specia**lC**haracteristics.measurement/1.0.0/Measurement.ttl_ to _io.catenax.specia**l_c**haracteristics.measurement/1.0.0/Measurement.ttl_.

Same here... please replace the camel case with an underscore. 😄

Reply:
done